public/data/sameem420.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "Muhammad Sameem",
+  "type": "personal",
+  "bio": "A JavaScript Dev (MERN STACK + REACT NATIVE) & Instructor. I am a philanthropist, tech & and open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects",
```

IMHO Having all caps in b/w of the text seems weird.

```suggestion
  "bio": "A JavaScript Dev (MERN Stack + React Native) & Instructor. I am a philanthropist, tech & and open-source lover. My dream is to give Free Education and build a community & platform that revolves around it. I always try to come up with unique and different project ideas that are for community use. Always open to collaboration on projects",
```